code/hispania/modules/mob/living/carbon/human/species/monkey/monkey_combat.dm

```diff
@@ -0,0 +1,494 @@
+/proc/IsLesserBeing(mob/living/carbon/human/H)
+	if(ismonkeybasic(H))
+		return TRUE
+	if(iswolpin(H))
+		return TRUE
+	if(isneara(H))
+		return TRUE
+	if(isfarwa(H))
+		return TRUE
+	if(isstok(H))
+		return TRUE
+	return FALSE
+
+/mob/living/carbon/human
+	var/aggressive=0 // set to 1 using VV for an angry monkey
+	var/frustration=0
+
+	var/melee_damage_lower = 4
+	var/melee_damage_upper = 10
+	var/attacktext = "bites"
+	var/attack_sound = 'sound/weapons/bite.ogg'
+	var/obj_damage = 0 //how much damage this simple animal does to objects, if any
+	var/armour_penetration = 0
+	var/melee_damage_type = BRUTE
+
+	var/pickupTimer=0
+	var/list/enemies = list()
+	var/mob/living/target
+	var/obj/item/pickupTarget
+	var/mode = MONKEY_IDLE
+	var/list/myPath = list()
+	var/list/blacklistItems = list()
+	var/maxStepsTick = 6
+	var/best_force = 0
+	var/martial_art = new/datum/martial_art
+	var/resisting = FALSE
+	var/pickpocketing = FALSE
+	//var/disposing_body = FALSE
+	//var/obj/machinery/disposal/bodyDisposal = null
+
+/mob/living/carbon/human/proc/IsStandingStill()
+	return resisting || pickpocketing //|| disposing_body
+
+// blocks
+// taken from /mob/living/carbon/human/interactive/
+/mob/living/carbon/human/proc/walk2derpless(target)
+	if(!target || IsStandingStill())
+		return FALSE
+
+	if(myPath.len <= 0)
+		myPath = get_path_to(src, get_turf(target), /turf/proc/Distance, 7 + 1, 250,1)
+
+	if(myPath)
+		if(myPath.len > 0)
+			for(var/i = 0; i < maxStepsTick; ++i)
+				if(!IsDeadOrIncap())
+					if(myPath.len >= 1)
+						walk_to(src,myPath[1],0,5)
+						myPath -= myPath[1]
+			return TRUE
```

Comment:
```suggestion
	if(myPath && myPath.len > 0)
		for(var/i = 0; i < maxStepsTick; ++i)
			if(!IsDeadOrIncap())
				if(myPath.len >= 1)
					walk_to(src,myPath[1],0,5)
					myPath -= myPath[1]
		return TRUE
```

Reply:
Se puede hacer este mismo cambio en los if después del for

Reply:
lol

Reply:
true dejame hacerlo